gapic/src/main/com/google/gapid/perfetto/models/QueryEngine.java

```diff
@@ -95,6 +107,24 @@ public QueryEngine(Client client, Path.Capture capture, StatusBar status) {
     });
   }
 
+  public ListenableFuture<ArgSet> getArgs(long id) {
+    return argsCache.get(id);
+  }
+
+  public ArgSet getArgsFromCache(long id) {
+    return argsCache.getIfPresent(id);
+  }
```

Comment:
Just curious, will this method going to be used in some scenarios in the future, when it is a soft cache?

Reply:
removed, since it's not actually used.